tests/test_migrations.py

```diff
@@ -3,9 +3,12 @@
 Tests for migrations, especially potentially risky data migrations.
 """
 
+from django.core.management import call_command
 from django.db import connection
 from django.db.migrations.executor import MigrationExecutor
-from django.test.testcases import TransactionTestCase
+from django.test.testcases import TestCase, TransactionTestCase
+from django.test.utils import override_settings
+from django.utils.six import StringIO
 
 
 class MigrationTestCase(TransactionTestCase):
```

Comment:
for real what is this even doing?

Reply:
hmm is it jist a base testcase? is it even subclassed by anything?

Reply:
Nope. Couldn't find this class referenced anywhere. 🤷

Reply:
hmm yeah I could test that out. It looks like tests are already failing so I'll figure out what that issue is and then give removal here a shot.

Reply:
seems like the tests don't fail with it removed so I'll probably just leave it removed in this commit.